apiserver/test/e2e/cluster_server_e2e_test.go

```diff
@@ -504,45 +503,39 @@ func TestDeleteCluster(t *testing.T) {
 	for _, tc := range tests {
 		tc := tc // capture range variable
 		t.Run(tc.Name, func(t *testing.T) {
-			actualRpcStatus, err := tCtx.GetRayApiServerClient().DeleteCluster(tc.Input)
+			actualRPCStatus, err := tCtx.GetRayAPIServerClient().DeleteCluster(tc.Input)
 			if tc.ExpectedError == nil {
 				require.NoError(t, err, "No error expected")
-				require.Nil(t, actualRpcStatus, "No RPC status expected")
+				require.Nil(t, actualRPCStatus, "No RPC status expected")
 				waitForDeletedCluster(t, tCtx, tc.Input.Name)
 			} else {
 				require.EqualError(t, err, tc.ExpectedError.Error(), "Matching error expected")
-				require.NotNil(t, actualRpcStatus, "A not nill RPC status is required")
+				require.NotNil(t, actualRPCStatus, "A not nill RPC status is required")
 			}
 		})
 	}
 }
 
 func createOneClusterInEachNamespaces(t *testing.T, numberOfNamespaces int) []*End2EndTestingContext {
 	tCtxs := make([]*End2EndTestingContext, numberOfNamespaces)
-	var wg sync.WaitGroup
-	wg.Add(numberOfNamespaces)
 	for i := 0; i < numberOfNamespaces; i++ {
-		go func(i int) {
-			defer wg.Done()
-			tCtx, err := NewEnd2EndTestingContext(t)
-			assert.NoError(t, err, "No error expected when creating testing context")
+		tCtx, err := NewEnd2EndTestingContext(t)
```

Comment:
A few functions requires execution inside of main goroutine:
- `tCtx.CreateComputeTemplate(t)`
- `tCtx.DeleteComputeTemplate(t)`